test/choosetests.jl

```diff
@@ -80,7 +80,7 @@ function choosetests(choices = [])
         prepend!(tests, linalgtests)
     end
 
-    net_required_for = ["socket", "parallel"]
+    net_required_for = ["socket", "parallel", "libgit2"]
```

Comment:
No, these tests are in libgit2, since they don't require a network connection but they do require `lo`. I suppose we could split them out yet again, but not sure it's worth it. @tkelman?